src/app/(dashboard)/residents/create/page.tsx

```
@@ -175,99 +196,108 @@
-                      <li key={field}>
-                        <strong>{field}:</strong> {error}
-                      </li>
-                    ))}
-                  </ul>
-                </div>
+      {/* Display validation errors if any */}
+      {Object.keys(validationErrors).length > 0 && (
+        <div className="mb-4 rounded-md bg-red-50 p-4 dark:bg-red-900/20">
+          <div className="flex">
+            <div className="flex-shrink-0">
+              <svg className="h-5 w-5 text-red-400" viewBox="0 0 20 20" fill="currentColor">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)